src/config.py

```diff
@@ -6,8 +6,11 @@
 
 DISCORD_KEY = cfg['discord']
 ART_CHANS = cfg['channels']['listening']
+MOD_CHANS = cfg['channels']['selfcurated']
```

Comment:
Minor: We could break this down further into cfg['channels']['art'] and cfg['channels']['mods'] for example. I'm fine how this is now tho.